bundles/org.eclipse.cdt.lsp.clangd/META-INF/MANIFEST.MF

```diff
@@ -33,6 +33,7 @@ Require-Bundle: org.eclipse.cdt.lsp;bundle-version="0.0.0",
  org.eclipse.ui;bundle-version="0.0.0"
 Bundle-Activator: org.eclipse.cdt.lsp.internal.clangd.editor.ClangdPlugin
 Service-Component: OSGI-INF/org.eclipse.cdt.lsp.clangd.BuiltinClangdOptionsDefaults.xml,
+ OSGI-INF/org.eclipse.cdt.lsp.clangd.DefaultClangdConfigurationVisibility.xml,
```

Comment:
I have barely used let alone reviewed this type of pattern - so I defer to @ruspl-afed to review this aspect. It looks ok to me, but I don't know what I could be missing.